src/Search.module.css

```diff
@@ -0,0 +1,25 @@
+.searchWrapper {
+    width: 100%;
+    display: flex;
+    justify-content: center;
+}
+
+.searchWrapper input{
```

Comment:
Given there is high style overlap with the input in `AddTodoForm`, think about attaching some of these directly to the `InputWithLabel` component to reduce repeating.